packages/datadog-plugin-graphql/src/resolve.js

```diff
@@ -9,40 +9,50 @@ class GraphQLResolvePlugin extends TracingPlugin {
   static get id () { return 'graphql' }
   static get operation () { return 'resolve' }
 
-  start ({ info, context, args }) {
+  start (fieldCtx) {
+    const { info, ctx: parentCtx, args } = fieldCtx
+
     const path = getPath(info, this.config)
 
+    // we need to get the parent span to the field if it exists for correct span parenting
```

Comment:
Isn't `parentCtx` already the context of the parent field? Why is a search needed if that's the case?

Reply:
Good catch, this `parentCtx` could be better described as the root context of the graphql operation, and corresponds to the root `graphql.execute` span. I have updated the naming to be `rootCtx` for this code. From the rootCtx, we still need the search to get the correct parent field. Hope that clears things up.

Reply:
Ah yeah it's way clearer now, I got confused the it was the direct parent without of the previous name.